layouts/container_footer.html

```diff
@@ -2,6 +2,6 @@
 
 <footer>
   <p class="pull-left">
-    &copy; Prometheus Authors 2016
+    &copy; Prometheus Authors 2017
```

Comment:
We use https://nanoc.ws/, a ruby static site generator. All HTML files are run through an [ERB filter](https://github.com/prometheus/docs/blob/master/Rules#L43).